src/components/Homepage.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { connect } from 'react-redux';
 import { Redirect } from 'react-router-dom';
 
-import conf from '../private/conf';
+import conf from '../private/conf-EXAMPLE';
```

Comment:
Do not include conf-EXAMPLE.js, it is for demonstration purposes only.
You should have two files in the /private folder, conf.js and conf-EXAMPLE.js.
conf.js is where the settings are specified, and it is .gitignored so that the information is kept safe.
conf-EXAMPLE.js can be then copy/pasted to conf.js for people who want to fork this repo.